src/style.css

```diff
@@ -0,0 +1,70 @@
+:root {
+  --background-image: url(./images/Search.svg) no-repeat 26px 25px;
+}
+
+body {
+  margin-top: 0;
+  margin-bottom: 0;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+.form {
+  margin-top: 20px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  font-weight: 900;
+  font-size: 16px;
+  line-height: 22px;
+  width: 100%;
+  height: 70px;
+  /* margin-top: 20px; */
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  background: var(--background-image);
+  padding-left: 62px;
+}
+
+.input--small {
+  font-size: 14px;
+  line-height: 19px;
+  height: 42px;
+  background-position: 13px 15px;
+  background-size: 11px;
+  padding-left: 33px;
+}
+
+.input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.input:focus {
+  border-radius: 3px;
+  background: var(--background-image),
+    linear-gradient(180deg, #fff 0%, #f6f6f7 100%);

```

Comment:
```suggestion
  linear-gradient(180deg, #fff 0%, #f6f6f7 100%);

```

Reply:
Lint expects the indentation of 4 spaces on this line and did not accept two